apps/mark/backend/src/app.ts

```diff
@@ -84,13 +85,33 @@ function buildApi(
       return auth.endCardlessVoterSession(constructAuthMachineState(input));
     },
 
-    getSystemSettings(): SystemSettings | undefined {
-      return workspace.store.getSystemSettings();
+    getElectionDefinition(): ElectionDefinition | null {
+      return workspace.store.getElectionDefinition() ?? null;
+    },
+
+    getSystemSettings(): SystemSettings | null {
+      return workspace.store.getSystemSettings() ?? null;
+    },
+
+    unconfigureMachine() {
+      workspace.store.setElection(undefined);
+      workspace.store.deleteSystemSettings();
+    },
+
+    configureSampleBallotPackage(): Result<
+      ElectionDefinition,
+      BallotPackageConfigurationError
+    > {
+      const electionDefinition = electionSampleDefinition;
+      const systemSettings = DEFAULT_SYSTEM_SETTINGS;
+      workspace.store.setElection(electionDefinition.electionData);
+      workspace.store.setSystemSettings(systemSettings);
+      return ok(electionDefinition);
     },
 
     async configureBallotPackageFromUsb(input: {
-      electionHash: string;
-    }): Promise<Result<void, BallotPackageConfigurationError>> {
+      electionHash?: string;
```

Comment:
Now that we have election def on the backend, it's no longer necessary to have the frontend provide election hash to the backend for auth checks. So we could just have the backend pull election hash directly.

That said, I know there's a separate comment thread on this PR about tackling things piecemeal, so that shift could def come in its own PR (I know I'd even said that I might tackle that). Just thought I'd note in case you wanted to start using the new approach here.

Reply:
Summarizing our chat, `electionHash` will never be defined on the auth object at this point (this endpoint is called to load election definition from USB so the machine can't yet have an election hash) so we can just construct a machine state with `{electionHash: undefined}` and remove the electionHash param.